packages/dart/pubspec.yaml

```diff
@@ -18,36 +18,40 @@ topics:
   - backend
 
 environment:
-  sdk: ">=3.2.6 <4.0.0"
+  sdk: ^3.6.2
```

Comment:
I'm unsure whether removing the upper bound is a good idea, but since even minor versions changes can be breaking, let's try.